style.css

```diff
@@ -112,11 +113,213 @@ body {
   background-color: #0056ec;
   color: #fff;
   border-radius: 4px;
-  padding: 18px 22px;
+  padding: 18px 30px;
   text-decoration: none;
   transition: opacity 0.5s;
 }
 
 .main__button:hover {
   opacity: 0.5;
+}
+  #checkbox {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/app.creation/pullRequest?prid=6278206)